### PR TITLE
Add update button in inquiries_list.xhtml

### PR DESCRIPTION
**Pull request by ManalJaidar**
**What's in the PR**
* The Button that updates inquiries in the user interface

### CODE REVIEW

File: src/main/webapp/WEB-INF/includes/c_configuration_interfaces.xhtml [hunk 1]
```diff
@@ -22,12 +23,27 @@
...
             <f:facet name="input">
                 <h:selectBooleanCheckbox id="ldmCachingEnabled"
                                          value="#{configurationBean.configurationMap[EnumConfiguration.LDM_CACHING_ENABLED]}"
-                                         styleClass="switch"/>
+                                         styleClass="switch"
+                                         onchange="storeConfiguration()"
+
+                >
+
+                <f:ajax execute="@form" listener="#{configurationBean.retune}" />
```
[the remark this answers is not in the document]
Comment (djuarezgf) [reply src/main/webapp/WEB-INF/includes/c_configuration_interfaces.xhtml:38]: (only for selectBooleanCheckbox)

[the remark this answers is not in the document]
Comment (djuarezgf) [reply src/main/webapp/WEB-INF/includes/c_configuration_interfaces.xhtml:38]: (not for text fields)

File: src/main/java/de/samply/share/client/control/ConfigurationBean.java [hunk 2]
```diff
@@ -144,8 +147,23 @@ public void setConfigurationTimingsMap(
...
   public boolean getAsBoolean(EnumConfiguration configuration) {
     return ConfigurationUtil.getConfigurationElementValueAsBoolean(configuration);
   }
 
+  /**
+   * Save automatically form information after any change.
+   *
+   * @param event event after modifying any form field.
+   */
+  public void retune(AjaxBehaviorEvent event) { // Note: the argument is optional.
```
Comment (TasDeniz) [inline src/main/java/de/samply/share/client/control/ConfigurationBean.java:165]: please change the method name
It doesnt give any information what this method is doing

**Review by TasDeniz — CHANGES_REQUESTED** [review]

**Review by TasDeniz — APPROVED** [review]

File: src/main/webapp/WEB-INF/includes/c_configuration_interfaces.xhtml (hunk 1 above)
[the remark this answers is not in the document]
Comment (djuarezgf) [reply src/main/webapp/WEB-INF/includes/c_configuration_interfaces.xhtml]: Done

**Pull request merged**